lunatik.h

```diff
@@ -327,6 +332,13 @@ do {									\
 	lua_pop(L, 1);							\
 } while (0)
 
+#define lunatik_optcfunction(L, idx, field, default_func)		\
+do {									\
+	(lua_getfield(L, idx, field) == LUA_TFUNCTION) ?		\
+		(void)0 :						\
+		(lua_pop(L, 1), lua_pushcfunction(L, default_func));	\
+} while (0)
```

Comment:
```suggestion
static inline void lunatik_optcfunction(L, idx, field, opt)
{
	if (lua_getfield(L, idx, field) != LUA_TFUNCTION) {
		lua_pop(L, 1);
		lua_pushcfunction(L, opt));
	}
}
```

I'm not fond of using the same name pattern as optinteger as we are not actually setting a struct field as pointed out, but I couldn't think on something better.. anyway, in this case, we don't need a macro.. an inline will do =). Please, move it to next optinteger anyway, it will be easier to maintain. Thanks.